src/bson/ext.rs

```diff
@@ -0,0 +1,259 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use mongodb::bson::{document::ValueAccessError, from_bson, from_document, Bson, Document};
+use sealed::sealed;
```

Comment:
ohh nice, I always do it ol' school style

Reply:
Would it be annoying to do it the old school way? I don't like pulling in dependencies if it can be avoided.

Reply:
🤷 I think it's cleaner this way.